include/openthread.h

```diff
@@ -1802,7 +1673,7 @@ OTAPI void OTCALL otPlatformReset(otInstance *aInstance);
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
  */
-void otFactoryReset(otInstance *aInstance);
+OTAPI void OTCALL otFactoryReset(otInstance *aInstance);
```

Comment:
It's the same way we are currently doing it for all the other functions. If you have a better suggestion, I'm open to suggestions.